openquake/commonlib/export/hazard.py

```diff
@@ -727,7 +727,7 @@ def export_gmf_txt(key, dest, sitecol, imts, ruptures, rlz,
     return {key: [dest]}
 
 
-def get_rup_idx(ebrup, etag):
+def get_eidx(ebrup, etag):
```

Comment:
Please make sure you want the name to be `get_eidx` rather than `get_eid`. Furthermore, in this diff I don't see any call to `get_eidx`. Please make sure calls to `get_rup_idx` are all renamed to `get_eidx` (or `get_eid`)